argus/backend/service/jenkins_service.py

```diff
@@ -15,9 +20,31 @@ class Parameter(TypedDict):
     value: Any
 
 
+class JenkinsServiceError(Exception):
+    pass
+
+
 class JenkinsService:
     RESERVED_PARAMETER_NAME = "requested_by_user"
 
+    SETTINGS_CONFIG_MAP = {
```

Comment:
What's the point of this mapping ? If they are identical?

Reply:
I was thinking if they were to diverge in the future - since iirc a job xml allows for more than one configured repo for example, so while this isn't really necessary right now (as evidenced that handling function used is the same, it's here in case we'll have to adapt the job advanced settings page further.